_includes/footer_custom.html

```diff
@@ -0,0 +1,10 @@
+<p>
+  {%- if site.email -%}
+  Questions? Be in touch with our team at <a class="u-email" href="mailto:{{ site.email }}">{{ site.email }}</a>
```

Comment:
Let's have this open in a new tab:

```suggestion
  Questions? Be in touch with our team at
  <a class="u-email" href="mailto:{{ site.email }}" target="_blank" rel="noreferrer noopener">{{ site.email }}</a>
```